fastjet/internal/delaunay/point.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay
+
+import (
+	"fmt"
+	"math"
+	"math/big"
+
+	"github.com/gonum/floats"
+)
+
+// Point in the X-Y Plane, that holds dynamic information about the
+// adjacent triangles, the nearest neighbor and the distance to that neighbor
+type Point struct {
+	// X and Y are the coordinates of the point.
+	X, Y float64
+	// adjacentTriangles is used for the triangulation, to find the nearest neighbor and
```

Comment:
`// adjacentTriangles is a list of triangles containing the point.` is enough - correct this if I have misinterpreted it.